### ISSUE TITLE
Boxed ```code``` text in plugin pages is too low contrast for accessibility

### ISSUE THREAD

**Issue by neuromusic** [chanzuckerberg/napari-hub#187]
*from June 2020 pre-launch bug bash*

**Created by**: Lia

## Description 

Ideally text should be black on the light grey (#f7f7f7) background for accessibility
https://www.figma.com/file/pDKU0CdSzpSWGaoXXQVJLz/v0-hub-final-designs?node-id=17%3A5

But if not possible to customize / not worth the time, then work with Lia to choose option from available presets to ensure we have one with higher contrast for a11y purposes.

- Also, ideally it should be spanning the full-width of the text (right now indented a bit), but I like the line-numbers!!!
- Also ideally would have squre corners (now slightly rounded)

## Info

**Bug source**: *unknown*

**Browser**: Chrome

**PR**: *none*

## Attachments

https://dl.airtable.com/.attachments/d64df194b972b7ce063b2a38f449e396/f3026377/ScreenShot2021-06-30at11_14_15AM.png
